queries.md

```diff
@@ -4,76 +4,96 @@
 
 ### 1. All the companies whose name match 'Babelgum'. Retrieve only their `name` field.
 
-<!-- Your Code Goes Here -->
+Filter: {name: "Babelgum"}
+Project: {name: 1, _id: 0}
 
 ### 2. All the companies that have more than 5000 employees. Limit the search to 20 companies and sort them by **number of employees**.
 
-<!-- Your Code Goes Here -->
+Filter: {"number_of_employees": {$gt: 5000}}
+Sort: {"number_of_employees": -1}
+Limit: 20
 
 ### 3. All the companies founded between 2000 and 2005, both years included. Retrieve only the `name` and `founded_year` fields.
 
-<!-- Your Code Goes Here -->
+Filter:{$and: [{founded_year: {$gte: 2000}}, {founded_year: {$lte: 2005}}]}
+Project: {"name": 1, "founded_year": 1, _id: 0}
```

Comment:
Aqui solo te falto entrar a tu arreglo para encontrar el valuation_amount, te dejo una solución: {$and: [{founded_year: {$lt: 2010}}, {"ipo.valuation_amount": {$gt: 100000000}}]} projection: {name:1, ipo:1, _id:0}